deployment/ansible/deploy.yml

```diff
@@ -58,7 +58,7 @@
           port: 3114
       chain:
         persist: true
-        db_path: "./db67"
+        db_path: "./db68"
```

Comment:
Would if be possible to add a line in release notes, for explaining DB incompatibility?

Reply:
No one has started on the release notes for 0.19 yet, right?